idea/src/org/jetbrains/kotlin/idea/inspections/NullChecksToSafeCallInspection.kt

```diff
@@ -78,7 +81,7 @@ private fun collectNullCheckExpressions(expression: KtBinaryExpression): Triple<
 
 private fun isNullChecksToSafeCallFixAvailable(expression: KtBinaryExpression): Boolean {
     val (lte, rte) = collectNullCheckExpressions(expression) ?: return false
-    return getDuplicates(lte, rte)?.parent as? KtDotQualifiedExpression != null
+    return lte.isChainStable() && getDuplicates(lte, rte)?.parent as? KtDotQualifiedExpression != null
```

Comment:
I don't know  how to write tests in this case because if i run test on code with unstable chain like this `fun main(args: Array<String>) {
    val a: Testtt? = Testtt()
    if (a?.a<caret> != null && a?.a.a != null) {
 
    }
}

class Testtt {
    val a: Testtt? = null
        get() = field
}`
compiler throws a error
`Smart cast to 'Testtt' is impossible, because 'a.a' is a property that has open or custom getter`.  What i must do in this case?